package.json

```diff
@@ -13,72 +13,72 @@
   },
   "author": "Edouard Richard",
   "dependencies": {
-    "classnames": "^2.2.5",
-    "jquery": "^2.2.3",
+    "classnames": "2.2.5",
+    "jquery": "2.2.4",
     "jquery-ui": "components/jqueryui#1.11.4",
-    "lodash": "^4.16.4",
+    "lodash": "4.17.4",
     "moment-timezone": "0.5.11",
     "nominatim-browser": "^2.0.2",
-    "prop-types": "^15.5.10",
-    "react": "^15.3.2",
-    "react-addons-shallow-compare": "^15.3.2",
+    "prop-types": "15.5.10",
+    "react": "15.5.4",
+    "react-addons-shallow-compare": "15.5.2",
     "react-bootstrap": "^0.30.5",
     "react-datepicker": "^0.41.1",
     "react-debounce-input": "^2.4.2",
-    "react-dom": "^15.3.2",
+    "react-dom": "15.5.4",
     "react-form-input": "^0.3.0",
     "react-geolookup": "^1.1.0",
     "react-moment-proptypes": "^1.4.0",
-    "react-redux": "^5.0.2",
+    "react-redux": "5.0.6",
     "react-select": "1.0.0-rc.5",
     "react-sortable-hoc": "^0.6.7",
     "react-textarea-autosize": "^5.0.7",
     "react-virtualized": "^9.8.0",
-    "redux": "^3.6.0",
+    "redux": "3.6.0",
     "redux-form": "^6.8.0",
     "redux-logger": "^2.7.0",
-    "redux-thunk": "^2.1.0",
+    "redux-thunk": "2.2.0",
     "reselect": "^2.5.4",
     "rrule": "^2.2.0",
-    "style-loader": "^0.13.1",
-    "superdesk-ui-framework": "superdesk/superdesk-ui-framework",
+    "style-loader": "0.18.2",
```

Comment:
I guess I've missed this one, it should go into devDependencies, but not a big deal. my idea was to keep everything that is needed for testing in dev and only put packages that must be installed in superdesk env in dependencies. the build stack (webpack/babel) should be provided by superdesk-client-core dependencies. makes sense?